### PR TITLE
Add/rpm stuff

### PR DESCRIPTION
**Pull request by AlvaroVega**
Fix part of issue: https://github.com/telefonicaid/sigfox-iotagent/issues/6

### CODE REVIEW

File: CHANGES_NEXT_RELEASE [hunk 1]
```diff
@@ -1,2 +1,3 @@
+add rpm stuff (spec file)
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE]: Issue #6 should be mentionced. "add" -> "Add"

File: rpm/SPECS/iotasigfox.spec [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+%config /etc/init.d/%{_service_name}
+%attr(755, root, root) /etc/init.d/%{_service_name}
+%config /etc/logrotate.d/logrotate-iotasigfox.conf
+%config /etc/iotasigfox.d/iotasigfox.default.conf
+%config /etc/cron.d/cron-logrotate-iotasigfox-size
+%config /etc/sysconfig/logrotate-iotasigfox-size
+%config /etc/sysconfig/iotasigfox.conf
+%{_install_dir}
+
+%changelog
```
Comment (fgalan) [inline rpm/SPECS/iotasigfox.spec]: As far as I understand, `%changelog` section has a predefined format that should be used. Have a look for instance to https://github.com/telefonicaid/fiware-orion/blob/master/rpm/SPECS/contextBroker.spec#L176

File: rpm/get_version_string.sh [hunk 3]
```diff
@@ -0,0 +1,125 @@
+#!/bin/bash
+#
+# Bash lib to know the RPM version and revision from a GitHub repository
```
Comment (fgalan) [inline rpm/get_version_string.sh]: Licence header missing.

File: rpm/SPECS/iotasigfox.spec [hunk 4]
```diff
@@ -0,0 +1,181 @@
+Summary: Sigfox IoT Agent
```
Comment (fgalan) [inline rpm/SPECS/iotasigfox.spec]: License header missing.

File: CHANGES_NEXT_RELEASE (hunk 1 above)
Comment (AlvaroVega) [reply CHANGES_NEXT_RELEASE]: CNR Updated in https://github.com/telefonicaid/sigfox-iotagent/pull/32/commits/e4426f8430cb50253af25f405db03638e5c7a27b

File: rpm/get_version_string.sh (hunk 3 above)
Comment (AlvaroVega) [reply rpm/get_version_string.sh]: All of the iotagents has the same missed header.

File: rpm/SPECS/iotasigfox.spec (hunk 4 above)
Comment (AlvaroVega) [reply rpm/SPECS/iotasigfox.spec]: Added license header in https://github.com/telefonicaid/sigfox-iotagent/pull/32/commits/c4b64018e59a70afef77312d670c787539d6d009

File: rpm/get_version_string.sh (hunk 3 above)
Comment (AlvaroVega) [reply rpm/get_version_string.sh]: Added license header in https://github.com/telefonicaid/sigfox-iotagent/pull/32/commits/c4b64018e59a70afef77312d670c787539d6d009

File: rpm/SPECS/iotasigfox.spec (hunk 2 above)
Comment (AlvaroVega) [reply rpm/SPECS/iotasigfox.spec]: Fixed in https://github.com/telefonicaid/sigfox-iotagent/pull/32/commits/b5e915f5bcbb0e2dbc99e9cab963460a477533d7

**Pull request merged**